src/CopyToLatest/targets/BranchInfo.props

```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Channel>release/3.0.1xx</Channel>
+    <Channel>master</Channel>
```

Comment:
Is this correct?

Reply:
Good catch.

Reply:
revert back to `<Channel>release/3.0.1xx</Channel>`. good catch indeed